docs/source/install/prerequisites.rst

```diff
@@ -205,30 +209,77 @@ Make sure that RabbitMQ is running with:
     Remove the mentioned option from the file ``/usr/libexec/rabbitmq/rabbitmq-env`` and restart the server.
     If you still have trouble getting RabbitMQ to run, please refer to the detailed instructions provided on the `website of RabbitMQ itself for generic Unix systems <https://www.rabbitmq.com/install-generic-unix.html>`_.
 
+
 .. _details_wsl:
 
 Windows Subsystem for Linux (Ubuntu)
 ====================================
 
-The guide for Ubuntu above can be followed but there are a few things to note:
+The guide for Ubuntu above can generally be followed, but there are a few things to note:
 
-#. The `windows native RabbitMQ <https://www.rabbitmq.com/install-windows.html>`_
-   should be installed and started,
-   since the version in WSL Ubuntu 18.04 does not work properly.
+#. It may be neccessary to install and start the `Windows native RabbitMQ <https://www.rabbitmq.com/install-windows.html>`_.
```

Comment:
Like this, this sentence is confusing - when is it necessary and when isn't it necessary?

Reply:
I agree. Unfortunately it reflects my ignorance as well: I am not sure when it is or isn't necessary. And neither does it seem are @zhubonan or @greschd as far as I can deduce from the previous discussion.
For me with my setup, this is not necessary, but it is for them with their setup.